python/lsst/ts/wep/task/calcZernikesTask.py

```diff
@@ -484,16 +488,20 @@ def run(
             ):
                 self.log.info("No donut stamps were selected.")
                 return self.empty(qualityTable=donutQualityTable)
+
         else:
+            self.log.info("Not running Donut Stamp Selector")
             donutQualityTable = QTable([])
+            selectedExtraStamps = self.stampsExtra
```

Comment:
Might be cleaner to just update `self.stampsIntra` and `self.stampsExtra` in lines 474-475 with a comment about updating them with selected donuts and remove these lines (495-496) and 499-500.

Reply:
I can't actually do that because then I'd break the tests that rely on the `self.empty` returned if no donuts get selected (eg. `maxSelect=0`, as the `self.empty` method needs the `stamps` metadata to create the `zkTable` metadata...  Updating `self.stampsIntra` afterwards works fine

Reply:
So I'll keep it as it was